centreon/www/front_src/src/Header/specs/Header.Poller.ts

```diff
@@ -0,0 +1,444 @@
+import { renderHook, act } from '@testing-library/react';
```

Comment:
If this is a test file, please rename it to `HeaderPoller.cypress.spec.tsx`

Reply:
the test is here `www/front_src/src/Header/specs/Header.cypress.spec.tsx` and imports all others sub-tests.